test/id.test.js

```diff
@@ -247,3 +247,64 @@ describe('mongodb default id name', function() {
     );
   });
 });
+
+describe('strictObjectIDCoercion', function() {
+
+  context('set to false (default)', function() {
+
+    var User = ds.createModel(
+      'User',
+      {
+        id: {type: Number, id: true},
+        name: String
+      }
+    );
+
+    beforeEach(function(done) {
+      User.deleteAll(done);
+    });
+
+    it('should find model with ObjectID type id', function(done) {
+
+      User.create({ name: 'John' }, function(err, customer) {
+          User.find({where: {seq: {inq: [customer.id]}}}, function(err, customers) {
+            customers.should.not.be.empty();
```

Comment:
@bajtos I am trying to test the current behavior, but something seem amiss. Am I doing something wrong here and expecting something else?

Reply:
> `{where: {seq: {inq: [customer.id]}}}`

Where is `seq` property coming from? Did you mean `id` instead?